app/views/notifications/_broadcast.html.erb

```diff
@@ -8,5 +8,9 @@
   </a>
   <div class="content notification-content broadcast-content">
     <%= json_data["broadcast"]["processed_html"].html_safe %>
+
+    <% if notification.json_data['broadcast']['type_of'] == "Welcome" %>
+      <div class="opt-out"><a href="/settings/notifications">Go to your settings to manage your notification settings.</a></<div>
```

Comment:
you could use `<%= user_settings_path(tab: :notifications) %>` here in lieu of the hardcoded path

Reply:
ohhhhh duh of course, i should've remembered to do that 🙈